microservice-app/src/main/java/com/sdevelopment/skt/microservice/dao/ProductRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.sdevelopment.skt.microservice.dao;
+
+import com.sdevelopment.skt.common.domain.Product;
+import org.springframework.data.repository.CrudRepository;
+
+public interface ProductRepository extends CrudRepository<Product, Long>, ProductRepositoryCustom {
```

Comment:
What is the purpose of `CrudRepository`? If you're using a repository, maybe it's not necessary to annotate your entity. You need to define your strategy to call the SP (by using your own implementation or Jpa)

Reply:
New strategy already defined and implemented.